abnormal-security/info.json

```diff
@@ -0,0 +1,1259 @@
+{
+  "name": "abnormal-security",
+  "label": "Abnormal Security",
+  "description": "Abnormal Security helps to prevent email and email-like attacks, automate your security operations and reduce your total spend with one extensible platform.",
+  "publisher": "Fortinet",
+  "cs_approved": false,
+  "cs_compatible": true,
+  "version": "1.0.0",
+  "category": "Email Security",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "title": "Server URL",
+        "name": "server_url",
+        "required": true,
+        "editable": true,
+        "visible": true,
```

Comment:
Tooltips are missing in Configuration fields.

Reply:
not required